application/F3DStarter.cxx

```diff
@@ -590,6 +617,7 @@ void F3DStarter::LoadFile(int index, bool relativeIndex)
 void F3DStarter::Render()
 {
   this->Internals->Engine->getWindow().render();
+  f3d::log::debug("Render done");
```

Comment:
Would be super cool if we add a timer here like `Render done in xx ms.`

Reply:
I dont really see the point ? we have the FPS indicator already. more over, this render method is not called every render, just the first before starting interaction.

Reply:
Yup, but when rendering offscreen, it could be useful. Anyway, can be done later.